hono/src/services/items.ts

```diff
@@ -24,7 +24,8 @@ const selectItemSchema = createSelectSchema(items, {
 })
 
 app.get('/', injectDB, async (c) => {
-  return c.json(await c.get('db').query.items.findMany())
+  await c.get('db').refreshMaterializedView(itemsExtended)
```

Comment:
You shouldn't have to call refresh on get (read only) methods. As long as the materialized view is refreshed on all write methods, we should be good to omit refreshing the entire view on reads.

Once this is removed, we do encounter another problem: what happens when we create this materialized view for the first time? The view isn't refreshed on the first read = the view isn't up to date until someone writes. You'll see this behavior reflected in the test cases. A good workaround would be to run `refreshMaterializedView` in the db migration script `hono/migrate.ts` and `hono/test/utils/db.ts`, after all migrations are complete.

Reply:
Good point! I will use this approach.